src/OPFS/OPFS.worker.js

```diff
@@ -165,6 +165,7 @@ async function fetchRGHUC(modelUrl) {
     return modelResponse
   } catch (error) {
     console.error('Error:', error)
```

Comment:
I think we should add Sentry tracking for cases like this, either by letting the error bubble or explicitly logging it with Sentry.captureException(err)